sdk/logz/azure-resourcemanager-logz/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+Please note, this package has been deprecated and will no longer be maintained after 01/01/2025. Refer to our deprecation policy (https://aka.ms/azsdk/support-policies) for more details.
```

Comment:
I don't think we need this part. This file only useful when there is new release.

Reply:
Addresses and removed this section.

Reply:
If you follow https://github.com/Azure/azure-sdk-for-java/blob/main/doc/deprecation_process.md and will do a release for this deprecation, you need to follow the description there (the line would be under the version e.g. `## 1.0.0-beta4`, and other things too like date)